libs/boost/include/boost/config/compiler/visualc.hpp

```diff
@@ -285,7 +285,7 @@
 
 //
 // last known and checked version is 19.00.22129 (VC14 Preview):
-#if (_MSC_VER > 1900 && _MSC_FULL_VER > 190022310)
+#if (_MSC_VER > 1800 && _MSC_FULL_VER > 190022310)
```

Comment:
i patched this manually to avoid a warning in visual studio, but it seems i forgot to add it in the apothecary script or probably i added but only to the visual studio target. if you are going to look into the ios thing, can you patch it back?